dcgtc/dgrants
# Refactor cart store + use grant metadata in cart

Closes #117, #121

- Moves all cart data into it's own store
- Grant metadata resolved from IPFS is now used on the cart page

To test
- Make sure cart and localStorage stay in sync when cart is modified on the grant registry page, the grant detail page, and the cart page
- Make sure grant names (e.g. "Test ipfs 1") are shown on the cart page
